Purchases/Public/RCPurchases.m

```diff
@@ -277,7 +273,6 @@ - (instancetype)initWithAppUserID:(nullable NSString *)appUserID
         self.identityManager = identityManager;
 
         self.notificationCenter = notificationCenter;
-        self.userDefaults = userDefaults;
```

Comment:
Was this intended?

Reply:
Ok I see that it's now only in the RCDeviceCache. Should it be removed from the arguments of this function then?

Reply:
good catch, `userDefaults` isn't needed here anymore